libtransmission/utils.cc

```diff
@@ -1134,7 +1160,10 @@ bool tr_moveFile(char const* oldpath, char const* newpath, tr_error** error)
 
         if (!tr_sys_path_remove(oldpath, &my_error))
         {
-            tr_logAddError("Unable to remove file at old path: %s", my_error->message);
+            tr_logAddError(fmt::format(
+                _("Unable to remove file at old path: {errmsg} ({errcode})"),
```

Comment:
Feels like this should be in line with all the other similar errors?..
```suggestion
                _("Couldn't remove '{path}': {errmsg} ({errcode})"),
                fmt::arg("path", oldpath),
```

Reply:
2b043b7796a45bdebbc5e356c7e4f7e1f5afc269